R/stochastic_process.R

```diff
@@ -273,17 +245,15 @@ process_scenario <- function(con, scenario, files, touchpoint,
   scenario_data$year <- scenario_data$year - scenario_data$age
   scen_coh <- agg_and_sort(scenario_data)
 
-  aggregated_data <- list(
+  names(scen_u5_coh)[names(scen_u5_coh) == "year"] <- "cohort"
+  names(scen_coh)[names(scen_coh) == "year"] <- "cohort"
+
+  list(
     u5_calendar_year = scen_u5_cal,
     u5_cohort = scen_u5_coh,
     all_calendar_year = scen_cal,
     all_cohort = scen_coh
   )
-
-  scenario_data <- NULL
-  gc()
```

Comment:
This was presumably added to flush memory? do we know if it is needed in the new code anywhere? Generally best if it's not needed :)